<launch>
    <!-- Automatically determine serial port to use: /dev/ttyACM0 (default) or /dev/ttyUSB0 (fallback) -->
    <arg name="serial_port" default="$(eval '/dev/ttyACM0' if eval('_' + '_import_' + '_(\'os\')').path.exists('/dev/ttyACM0') else '/dev/ttyUSB0')"/>

    <include file="$(find rallycar)/launch/include/static_tf.launch"/>
    <include file="$(find rallycar)/launch/include/lidar.launch"/>
    <node pkg="rallycar" name="rallycar_driver" type="rallycar_driver_node" output="screen">
        <param name="serial_port_fd" value="$(arg serial_port)"/>
        <param name="serial_port_baudrate" value="115200"/>
    </node>
</launch>
